src/stores/catastrophes.ts

```diff
@@ -12,11 +12,14 @@ export const useCatastropheStore = defineStore('catastropheStore', {
         };
     },
     getters: {
-        findCatastrophes: state => (year: number, district = 0) => {
+        findCatastrophes: state => (year: number, district = 0, typeFilter = '') => {
             let catastrophes = state.catastrophes.get(year) ?? List();
             if (district) {
                 catastrophes = catastrophes.filter(x => x.district === district);
             }
+            if (typeFilter) {
+                catastrophes.filter(x => x.type === typeFilter);
```

Comment:
`filter`, que ça soit sur un array ou une liste immuable, ça filter pas in place, ça renvoie la nouvelle collection filtrée, faque en ce moment tu filtres absolument rien.

Reply:
Omg merci